public/tpl/header.html

```diff
@@ -128,6 +128,52 @@
                         </li>
                     </ul>
                 </li>
+                <li class="menu-dropdown classic-menu-dropdown ">
+                    <a data-hover="megamenu-dropdown" data-close-others="true" data-toggle="dropdown" href="javascript:;"> Community
+                        <i class="fa fa-angle-down"></i>
+                    </a>
+                    <ul class="dropdown-menu pull-left">
+                        <li>
+                          <a href="https://forum.ethereumclassic.org">
+                              <i class="fa fa-group"></i>
+                              <span class="title" data-toggle="tooltip" title="Community Chat"> The Forum</span>
+                          </a>
+                        </li>
+                        <hr>
+                        <li>
+                            <a href="https://tokenmint.io">
+                                <i class="fa fa-leaf"></i>
+                                <span class="title" data-toggle="tooltip" title="ERC223 Token Factory" > TokenMint.io</span>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="http://classicetherwallet.com">
+                                <i class="fa fa-folder-open"></i>
```

Comment:
let's make this one go to https://ethereumproject.github.io/etherwallet/ because classicetherwallet.com forwards there anyway